clusterloader2/pkg/measurement/common/wait_for_pvcs.go

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
High-level comment:
The review latency of a code is exponential to its size. So more you split it into smaller chunks, the faster you will have it done.